src/Sarif.Viewer.VisualStudio.Core/Telemetry/KeyEventTelemetry.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+
+using Microsoft.Sarif.Viewer.Shell;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.Sarif.Viewer.Telemetry
+{
```

Comment:
Please add doc comments for the class and all public methods

Reply:
added